pkg/auth/server/tokenrequest/endpoints.go

```diff
@@ -48,11 +48,21 @@ func (endpoints *endpointDetails) requestToken(w http.ResponseWriter, req *http.
 func (endpoints *endpointDetails) displayToken(w http.ResponseWriter, req *http.Request) {
 	w.Header().Set("Content-Type", "text/html")
 
+	header := `
+<style>
```

Comment:
guess it doesnt have to be a template since there are no substitutions, but at least make it a const instead